src/constants/multiplierBadges.ts

```diff
@@ -14,40 +15,53 @@ export const multiplierBadges: MultiplierBadge[] = [
     icon: '🧪',
     color: '#6000FF',
     multiplier: 1.25,
-    link: 'https://blog.ambire.com/announcing-the-wallet-token-a137aeda9747'
+    link: 'https://blog.ambire.com/announcing-the-wallet-token-a137aeda9747',
+    icon_svg: 'rewards-beta-tester'
```

Comment:
I see no point in adding `icon_svg` in the common codebase.

1. It's web app specific, so why in common code?

1. On the web app you can wire-up these icons by their `id`. Why introduce another prop?

Reply:
1. Unfortunately, no. Web SVGs don't work out of the box for native apps.

2. Okay, I see. Please either use "id" or map the `icon_svg` prop in ambire-wallet instead.